src/test/ReactTestUtils.js

```diff
@@ -269,9 +269,8 @@ var ReactTestUtils = {
       componentType
     );
     if (all.length !== 1) {
-      throw new Error(
-        'Did not find exactly one match for componentType:' + componentType +
-        ' (found ' + all.length + ')'
+      throw new Error('Did not find exactly one match ' +
```

Comment:
@spicyj For the error messages of `findRenderedDOMComponentWithClass` and `findRenderedDOMComponentWithTag`, there's no newline after the paren. Should I still go with a newline after the opening paren?